cmd/transportRequestUploadCTS.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/SAP/jenkins-library/pkg/command"
 	"github.com/SAP/jenkins-library/pkg/log"
 	"github.com/SAP/jenkins-library/pkg/telemetry"
-	"github.com/SAP/jenkins-library/pkg/transportrequest"
+	transportrequest "github.com/SAP/jenkins-library/pkg/transportrequest/cts"
```

Comment:
how about not renaming it to transportrequest? Then we would see the issue codeclimate is pointing out.

Reply:
.. but that was the smallest change I found.
And it would be possible to refer always to "transportRequest" for all the upload types.
But that is not really important. I'm not against renaming it.

Still think the cc comment is questionable since there is nothing wrong with using aliases ...

Reply:
Just now came in my mind:

* In groovy we have one upload step bundeling all 3 types together (SOLMAN, CTS, RCF). Over time the coding for these types got more and more specific to the particular type. Hence I start with 3 independent steps here.
* Maybe we have to provide one common step, forwarding to the particular steps in order to make the shift from groovy to go easier.
* In case that would happen we have to use aliasing since the 3 packages would be used in that common class. Right now aliasing is not strongly required, it is a more the less "fancy" approach in order to keep the diff small.  But in that case it would be mandatory. Don't know it is easier to understand when we still have the names as they are now.
* On the other hand we can choose aliases so it gets clear from the alias which flavour we have ...

Reply:
Aliasing is fine. But could you maybe change the cts method names as cc suggests? For example Connection instead of CTSConnection.

Reply:
As already outlined elsewhere IMO the cc comments are questionable since the name are not that precise when aliasing is used. On the other hand I don't have a bigger problem with renaming --> done.

@o-liver : please re-check